### DISCUSSION TITLE
update on svchost mitigation issue?

### DISCUSSION THREAD

**Discussion by agpt8** [HotCakeX/Harden-Windows-Security#194] [Q&A] [answered]
A while ago, it was found that the svchost mitigation causes some MS unsigned dlls pollute event log unnecessarily and causes high resource usage.

Details on this can be found on this pull request: #158

Feedback for this was also filed but it does not show any updates from MS's side: https://aka.ms/AAnaa72, https://aka.ms/AAnakja, https://aka.ms/AAnp88q

While there is no official word from MS, have they made any changes to the associated files silently in any updates released after?

Currently, I have to go in and turn off svchost mitigation each time I run the Harden-Windows-Security module. Any updates on this?

**Comment by HotCakeX** [comment, marked as answer]
Hi,
They signed the file and it's been signed for a while now, unfortunately I don't know exactly when they released it to the stable channel but judging by the signing date it actually looks like they signed it the day after Harden windows security module v.0.2.6 was released!

<img width="1280" alt="image" src="https://github.com/HotCakeX/Harden-Windows-Security/assets/118815227/e9ef346b-3b4c-4afc-abc8-5be5f7c19d25">

<br>

<br>

I was going to mention it in the next update's release notes too: https://github.com/HotCakeX/Harden-Windows-Security/pull/193

**Reply by agpt8** [reply]
Oh this is great! Thank you for the update. I'll wait for the new release then.

**Reply by HotCakeX** [reply]
No problem, sounds good :)
